database/migrations/2021_05_14_173108_repairdir_role.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class RepairdirRole extends Migration
```

Comment:
Hah, this raises an interesting question about migrations from the Repair Directory that change Restarters.  We should chat about it at retro.